modules/rgbd/samples/large_kinfu_demo.cpp

```diff
@@ -127,7 +127,7 @@ int main(int argc, char** argv)
     // These params can be different for each depth sensor
     ds->updateParams(*params);
 
-    cv::setUseOptimized(true);
+    cv::setUseOptimized(false);
```

Comment:
This will be enabled as soon as OCL implementation works, right?

Reply:
yes